two_replikas_chat.py

```diff
@@ -100,8 +92,13 @@ def get_most_recent_response(browser):
 #Insert start text in rep 2
 def type_most_recent_response(browser, response):
     text_box = browser.find_element_by_id("send-message-textarea")
-    response = remove_emojis(response) #Remove emojis if using Chrome (Chromedriver can't process)
-    text_box.send_keys(response)
+
+    # Mod: Workaround for emoji problem
+    script="var elm = arguments[0],txt=arguments[1];elm.value += txt;"
+    browser.execute_script(script, text_box, response)
+
+    # Mod: neccessary else send_keys throws an error
+    text_box.send_keys(" ")
```

Comment:
Sending the response to the textarea via javascript. The empy space in line 101 is needed to dispatch the native events so that the send works. Simpler this way than to dispatch the events in javascript.